spec/integration/integration_spec.rb

```diff
@@ -627,6 +627,60 @@ class Foo < Comment
       end
 
       describe 'undo' do
+        [nil].each do |method|
```

Comment:
Collapse tests for `undo` and `undo!` together, maybe with two loops, one over method/instance and inside over `undo!` and `undo!`. I also would like an explicit test that `undo!` is just `undo` + `save!`.